src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+  height: 16px;
+  justify-content: space-between;
+  margin-bottom: 24px;
+
+  &--4 {
+    gap: 4px;
+  }
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    background-position: center;
```

Comment:
The `background-position` property should be combined with the `background` shorthand property used in lines 12 and 24 for better code organization.